plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -412,11 +416,15 @@ def convert_pyr_to_tensor(self, pyr_coeffs):
         ----------
         pyr_coeffs: `OrderedDict`
             the pyramid coefficients
```

Comment:
we talked about this a bit in person -- should have more detail in the docstring about how batches, channels, real/imag are all interwoven.